config.json

```diff
@@ -639,7 +639,10 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "recursion",
+        "strings",
+        "pattern_recognition",
+        "text formatting"
```

Comment:
Please change to `text_formatting` for consistency with https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt